redis/client.py

```diff
@@ -3182,7 +3203,9 @@ def _zaggregate(self, command, dest, keys, aggregate=None):
         if aggregate:
```

Comment:
Given the multiple functions that now use _zaggregate - before merging them all, we should validate the [aggregate option value](https://redis.io/commands/zunionstore).

Reply:
done